### ISSUE TITLE
Bonded small amount of BSQ sellers

### ISSUE THREAD

**Issue by MwithM** [bisq-network/growth#243]
To make it easier for small volume traders, BSQ trading started with a different security model, based on bonds from sellers. See [wiki](https://bisq.wiki/Buy_BSQ#Buying_small_amounts) for details.
The first post of this issue will be updated reflecting the current valid bonds for sellers, I'll keep it updated.

- @mwithm locked 150BSQ in [txid](https://mempool.emzy.de/es/bisq/tx/b9830a33c3c17c8db1de1674c8388e79c2129c4ca2312bcc0772b1d03fdac037) until block 687531 (90 days since 14 March).

**Comment by pazza83** [comment]
Hi @MwithM is there a wiki about bonds? I have looked but cannot find anything.

I get a little confused by them. 

For instance your bond of 150 BSQ. I can see it on the DAO > Bonding > Bonds. But it would be great if when a user made the bond they can link it to a GitHub issue like on the compensation requests. This would identify the user as the bond holder and prevent a scammer claiming someone else's bond as their own.

**Comment by MwithM** [comment]
No, there is nothing at the Wiki yet. This would be the github issue for linking the bonds, which should be updated by me on regular updates for the first comment and the keybase group headline. There's no need to create a new issue. New bonded sellers could contact me now before bonding, to help each other to do it right.
I want to get more familiar with how bonds work myself before creating a wiki page, that's why this matter at the wiki is so vague.

**Comment by flix1** [comment]
What a great idea! I'm will try to create a bond as well.

**Comment by MwithM** [comment]
I still find missing a way to identify this bond with your identity, other than you communicated in advance the creation of this bond.
There should be something you "reveal" to us that only you can. I see the salt on my bond, but I don't know what would you gain by knowing it, there must be a way to verify that.

I have updated this first post and will update #buy-bsq channel header. Do you know a simple blockchain "calendar", or calculator to know ETA for block 691272?

**Comment by flix1** [comment]
> Do you know a simple blockchain "calendar", or calculator to know ETA for block 691272?

No. But 10.000 blocks is ≈ 100.000 minutes -> 1.666 hours -> 69 days.

**Comment by stopfeds** [comment]
I also just bonded 100 BSQ to try out my first BSQ/Lightning trade on Keybase.

My offer: 20 BSQ for 90k sats
keybase://chat/bisq#buy-BSQ/213

["Proof of bond"](https://mempool.space/de/bisq/tx/9b2eab26ecc5921c52954c774831dd3896c4edacf9f364ce1fb06c7d757c12f3)

**Comment by pazza83** [comment]
I think it would be good to develop the bonds.

Maybe some of the concerns I have come from a lack of understanding, but generally I think a bond without identification has limitations.

My concerns are:

- Reputation bonds do not link to GitHub / Keybase identity
- The user providing the reputation bond cannot with 100% certainty link the bond to their identity
- If a user's bond is to be burned then burning the bond the user provided means that stakeholders can not be certain they are voting to burn the bond of that user
- The way it is at present does not scale well

Here are a couple of sceenshots from the Bisq DAO section:

![proposal for boned role](https://user-images.githubusercontent.com/66367547/116832000-b25a8980-abaa-11eb-8b74-2be88cbc2a54.PNG)
Proposal for bonded role - includes a reference to Github identify 


![Reputation bonds](https://user-images.githubusercontent.com/66367547/116832014-c7cfb380-abaa-11eb-8975-390392edf6f0.PNG)
Reputation bond - no reference to Github and/or Keybase identity


Would adding Github and/or Keybase identity to the Reputation bond be simple to add? It seems like a fix to the issues I raised.

**Comment by MwithM** [comment]
Or revealing a secret here, that only the bond submitter could know, and we verify it. I thought that the purpose for the "salt" was that.

**Comment by pazza83** [comment]
Thanks I was not aware the SALT could be used for proof. Still think having the option to add Keybase / Gitbub identity would be good

**Comment by chimp1984** [comment]
The salt is the pre-image for the hash. Only the bond creator knows that, but once shared anyone knows it, so it is a one-time usage security feature. 
Better would be to add the siging tool we have for prrof of burn. Then the owner can always sign a challenge message and by that proof that he is the owner. As long the challenge message is choosen by the challenger (trade peer) and sufficient random it is safe. I think technically it should work very similar like proof of burn, so its mostly UI copy&paste work to add that feature to bonded reputation as well. With that anyone could use Bisq to do the verification of one who claims to have a bonded reputation.

**Comment by MwithM** [comment]
I don't understand what's the deal about making public the salt one time, and reflecting that the one who knows the salt is the bond owner, as that bond will be used as long as it's locked for the same trader. But I don't know how would someone verify that the salt corresponds to a locked bond.

**Comment by MwithM** [comment]
My [bond](https://mempool.space/de/bisq/tx/9b2eab26ecc5921c52954c774831dd3896c4edacf9f364ce1fb06c7d757c12f3) should have been unlocked, I have to manually claim the funds? This txid does not appear under `bonded roles`
 tab but it's still displayed at `my reputation bonds` and at `dashboard - locked in bonds`.
![imagen](https://user-images.githubusercontent.com/50149324/123064669-c5e9da00-d40e-11eb-93a1-3dc938eb54c8.png)
